pegasus/helpers/section_api_helpers.rb

```diff
@@ -485,6 +485,7 @@ def script
   def to_owner_hash
     to_member_hash.merge(
       script: script,
+      course_id: @row[:course_id],
```

Comment:
Caching courses does seem like something we should be doing. I'll put it on my list.